FEMComputationalFluidDynamicsAnalysis/postProcessing/computeResultingTotalForceOnSelectedDomain.m

```diff
@@ -24,16 +24,11 @@
 %
 %% Function main body
     
-% Reshape the vector to [Vx, Vy, p]
-FComplete = -reshape(FComplete, [analysis.noFields,                     ...
-             length(FComplete) / analysis.noFields])';
-
-% Select only the forces on the domain
-Force = FComplete(nodesDomain, 1:analysis.noSpatialDimensions);
-
 F = zeros(analysis.noSpatialDimensions,1);
+DOFsDomain = analysis.noFields * nodesDomain - analysis.noSpatialDimensions;
```

Comment:
In our case analysis.noFields = 3 and analysis.noSpatialDimensions = 3 so if nodesDomain contains node with id 1 then DOFsDomain will have a 0, I am surprised this even works.